slicer-build-base/update.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import subprocess
+
+#
+# Usage:
+#
+#   $ ./update.py <svn_revision> <svn_revision_date>
+#
+# Example:
+#
+#   $ ./update.py 25199 2016-06-19
+#
+
+def update_dockerfile(dockerfile, svn_revison, svn_revision_date):
```

Comment:
Perhaps this could warn or require an override parameter if the revision or date is earlier than what's currently in dockerfile.